pkgs/by-name/pi/picom-pijulius/package.nix

```
@@ -0,0 +1,19 @@
+{ picom, lib, fetchFromGitHub }:
+picom.overrideAttrs (oldAttrs: rec {
```

```suggestion
picom.overrideAttrs (finalAttrs: previousAttrs {
```

Use finalAttrs instead of rec